commons/src/enums/Nylas.ts

```diff
@@ -20,3 +20,8 @@ export enum MessageType {
   DRAFTS = "drafts",
   MESSAGES = "messages",
 }
+
+export enum MailboxThreadClickAction {
+  DEFAULT = "default",
+  CUSTOM = "custom",
+}
```

Comment:
This might be outside the scope of your PR, but could you add "mailbox" to this, then it can be used in the email component? https://github.com/nylas/components/blob/main/components/email/src/Email.svelte#L74

Reply:
Email already has a `click_action` prop though.